src/clarity-angular/select/select-input.ts

```diff
@@ -0,0 +1,91 @@
+import {NgModule} from "@angular/core";
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, ElementRef, EventEmitter, HostListener, Input, OnDestroy, Output} from "@angular/core";
+import {Observable} from "rxjs/Observable";
+import {Subscription} from "rxjs/Subscription";
+
+import {IfOpenService} from "./../utils/conditional/if-open.service";
+import {RootSelectService} from "./providers/select.service";
+import {Select} from "./select";
+
+
+@Directive({selector: "[clrSelectInput]"})
```

Comment:
In the link I posted in my previous review about ARIA roles, they show a full example. You added the listbox and options roles, but you're still missing the `combobox` role, `aria-owns`, `aria-autocomplete`, `aria-controls`, ...

Here is the example from the W3 docs:
```html
<div aria-label="Tag" role="combobox" aria-expanded="true" aria-owns="owned_listbox" aria-haspopup="listbox">
    <input type="text" aria-autocomplete="list" aria-controls="owned_listbox" aria-activedescendant="selected_option">
</div>
<ul role="listbox" id="owned_listbox">
    <li role="option">Zebra</li>
    <li role="option" id="selected_option">Zoom</li>
</ul>
```